src/actions/actions.py

```diff
@@ -20,8 +20,8 @@ class MakeshiftAction:
 
 
 makeshift_actions = [
-    MakeshiftAction("define", "editor.action.revealDefinition"),
     MakeshiftAction("drink cell", "jupyter.insertCellAbove"),
+    MakeshiftAction("follow", "editor.action.revealDefinition"),
```

Comment:
Personally I use reveal definition much more than I use find. Maybe we should rename find instead?